bedrock/firefox/templates/firefox/includes/download-button.html

```diff
@@ -43,7 +43,9 @@
       <li class="os_{{ plat.os }}{% if plat.arch %} {{ plat.arch }}{% endif %}">
         <a class="download-link"
            href="{{ plat.download_link }}"{% if plat.download_link_direct %}
-           data-direct-link="{{ plat.download_link_direct }}"{% endif %}>
+           data-direct-link="{{ plat.download_link_direct }}"{% endif %}
+           {% if plat.os == 'android' %}data-link-type="download" data-download-os="Android"
+           {% else %}data-link-type="download" data-download-os="Desktop"{% endif %}>
```

Comment:
This is incorrectly adding `data-download-os="Desktop"` to iOS download links as well as desktop.